main_page.py

```diff
@@ -0,0 +1,71 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+import my_account_page
+
+
+class Constant_newsletter:
```

Comment:
```suggestion
class ConstatntNewsletter:
```
Nazwy klas powinny być ConstantNewsletter https://peps.python.org/pep-0008/#class-names tak samo w reszcie kodu